.github/workflows/test-iso.yml

```diff
@@ -0,0 +1,57 @@
+name: Test Generate ISO
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths-ignore:
+      - 'Containerfile'
+      - '*.md'
+      - 'LICENSE'
+      - 'CODEOWNERS'
+
+jobs:
+  build-and-push-iso:
+    name: Build ISO
+    runs-on: ubuntu-latest
+    container:
+      image: fedora:39
+      options: "--privileged"
+    permissions:
+      contents: read
+      packages: write
+    strategy:
+      fail-fast: false
+      matrix:
+        version: 
+          - 38
+          - 39
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Build ISO
+        uses: ./
+        with:
+          ARCH: 'x86_64'
+          IMAGE_NAME: 'base-main'
+          IMAGE_REPO: 'ghcr.io/ublue-os'
+          VARIANT: 'Kinoite'
```

Comment:
The image name and variant appear to conflict here. For Kinoite I believe it should be `kinoite-main`

Reply:
Kinoite is being used by the installer ISO itself, not for base-main. To build base-main (which is our smallest image and fastest to test), it needs to use the Kinoite variant for the installer to allow the user to set a password upon installing. Does that make sense?

Reply:
Yeah, definitely. Very strange and interesting workaround but if it works it works :)